### ISSUE TITLE
totally separate scalling for handy note icons and handy notes icons refresh

### ISSUE THREAD

**Issue by Cherioz** [mikepauer/Carbonite#594]
now Icon scalling for handy notes is tied to some other scalling attributesw unless i am missing something, if tyhe handy note scalling is seperate would help a lot to make it clear on map rather than being either too small to see or too big dpensant on other setting for icons , if you know optimal setting i am happy to try

Second and more important when using handynotes on carbonite , when i complete a treasure and its done, the minimap shows its done and remove icon .. but carponite need a reload to show it !!!! , so in sesssion i see treasurs up on map that i completed!! after relead they are and reflected correctly

**Comment by IrcDirk** [comment]
Yep i now about those. No time for this. But this will be fixed eventually.
